runtime/compiler/optimizer/InterpreterEmulator.cpp

```diff
@@ -518,7 +518,10 @@ InterpreterEmulator::isCurrentCallUnresolvedOrCold(TR_ResolvedMethod *resolvedMe
    {
    bool isIndirectCall = false;
    if (current() == J9BCinvokevirtual)
+      {
       isIndirectCall = true;
+      return ((isUnresolvedInCP && !resolvedMethod) || (resolvedMethod && resolvedMethod->isCold(comp(), isIndirectCall)));
```

Comment:
Can you just explain the subtle change being made here? it isn't immediately obvious why this is changing?

Reply:
copied from the legacy code. On second thought, I think it is safe to just delete it.